components/Cards/tests/snapshot/MDCCardSnapshotTests.m

```diff
@@ -13,24 +13,92 @@
 // limitations under the License.
 
 #import "MDCSnapshotTestCase.h"
+#import "MaterialCards+Theming.h"
 #import "MaterialCards.h"
 
 @interface MDCCardSnapshotTests : MDCSnapshotTestCase
 
+@property(nonatomic, strong) MDCSemanticColorScheme *colorScheme;
```

Comment:
Shouldn't the container scheme hold the color scheme property?

Reply:
Actually yeah that is better. Will make that change